src/psyclone/dynamo0p3.py

```diff
@@ -5700,7 +5827,7 @@ def _cell_ref_name(self):
         '''utility routine which determines whether to return the cell value
         or the colourmap lookup value '''
         if self._kern.is_coloured():
-            return "cmap(colour, cell)"
+            return self._kern.colourmap + "(colour, cell)"
```

Comment:
Please document the associated method arguments (for _cell_ref_name)

Reply:
Done.